test/components/form-version/view-xml.spec.js

```diff
@@ -0,0 +1,12 @@
+import FormVersionViewXml from '../../../src/components/form-version/view-xml.vue';
+import { mount } from '../../util/lifecycle';
+
+describe('FormVersionViewXml', () => {
+  it('formats the XML', () => {
+    const modal = mount(FormVersionViewXml, {
+      propsData: { state: true },
+      requestData: { formVersionXml: '<x><y/></x>' }
+    });
+    modal.first('code').text().should.equal('<x>\r\n    <y/>\r\n</x>');
```

Comment:
`\r\n`? That's what the library does?

Reply:
Yes! It was also surprising to me. It's possible to specify something else, but `\r\n` is the default and seems to render OK in the browser.